ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -205,15 +205,13 @@ public struct AvatarGroup: View {
 
                 let ringPaddingInterspace = nextAvatarHasRing ? interspace - (ringOffset + ringOuterGap) : interspace - ringOffset
                 let noRingPaddingInterspace = nextAvatarHasRing ? interspace - ringOuterGap : interspace
-                let rtlRingPaddingInterspace = (nextAvatarHasRing ? -x - ringOuterGap : -x + ringOffset)
-                let rtlNoRingPaddingInterspace = (nextAvatarHasRing ? -x - ringOffset - ringOuterGap : -x)
                 let stackPadding = (currentAvatarHasRing ? ringPaddingInterspace : noRingPaddingInterspace)
 
+                let cutoutSize = isLastDisplayed ? (ringOuterGap * 2) + imageSize : nextAvatarSize
```

Comment:
This needs to consider 3 tokens for positioning the avatar:
- ringInnerGap
- ringThickness
- ringOuterGap

It may work today because the values are identical, but it should consider the tokens independently.

Reply:
And I believe this needs to be fixed in the other lines too.

Reply:
I think for this line ringOuterGap is the value we want. I think the idea here is to say that the last avatar is the overflow, and so it should be ringOuterGap + imageSize + ringOuterGap